src/xo-app/host/tab-network.js

```diff
@@ -117,57 +128,143 @@ class PifItem extends Component {
     }
     return reconfigurePifIp(this.props.pif, { mode })
   }
+
   _onEditIp = () => this._configIp('Static')
```

Comment:
If the only way you'll call `_configIp` is with "Static" as param, you don't need the whole `_configIp` method. Just extract what you need.